packages/techdocs-cli/src/commands/index.ts

```diff
@@ -44,6 +44,50 @@ export function registerCommands(program: CommanderStatic) {
     .alias("build")
     .action(lazy(() => import("./generate/generate").then(m => m.default)));
 
```

Comment:
Not sure, should we include this new command iat the docs here?
https://github.com/backstage/techdocs-cli/tree/mob/migrate-command/docs#generate-techdocs-site-from-a-documentation-project